packages/manager/modules/web-paas/src/details/service/add-addon/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,23 @@
+{
+  "web_paas_service_add_addon_title_storage": "Augmentez votre stockage",
+  "web_paas_service_add_addon_title_environment": "Augmentez vos environnements",
+  "web_paas_service_add_addon_title_license": "Augmentez votre licences utilisateur",
+
+  "web_paas_service_add_storage_description": "Vous possédez actuellement {{ storage }} Go de stockage additionnel. Le prix de votre stockage additionnel est lié au nombre de vos environnements.",
+  "web_paas_service_add_license_description1": "Vous avez souscrit à l’offre {{ offerName }}. Vous avez {{ maxLicences }} licences utilisateur inclues et {{ license }} ",
+  "web_paas_service_add_license_description_licence_singular": "licence utilisateur additionnelle.",
+  "web_paas_service_add_license_description_licence_plural": "licences utilisateur additionnelles.",
+  "web_paas_service_add_license_description2": "Vous pouvez ajouter des licences utilisateur pour permettre à vos collaborateurs d’accéder à vos projets Web PaaS.",
+  "web_paas_service_add_staging_description": "Vous possédez actuellement {{ environment }} environnements. Le nombre de vos environnements impacte également le prix de votre stockage qui est appliqué à tous vos environnements.",
+
+  "web_paas_service_add_addon_prorata_price_environment": "Le prorata à payer sera de {{ prorataPrice }} € pour le mois de {{ currentMonth }} et de {{ nextMonthPrice }} €/mois pour les prochains mois sur vos nouveaux environnements. Le prix total de votre stockage additionnel en fonction du nombre d’environnement sera de {{ totalPrice }} €/mois.",
+  "web_paas_service_add_addon_prorata_price_storage": "Le prorata à payer sera de {{ prorataPrice }} €  pour le mois de {{ currentMonth }} et de {{ nextMonthPrice }} €/mois pour les prochains mois pour votre nouveau stockage additionnel. Soit un total de {{ totalPrice }} €/mois calculé en fonction du nombre de vos environnements.",
+
+  "web_paas_service_add_addon_prorata_price_license": "Le prorata à payer sera de {{ prorataPrice }} € pour le mois de {{ currentMonth }} et de {{ nextMonthPrice }} €/mois pour les prochains mois sur vos nouvelles licences utilisateur supplémentaires.",
```

Comment:
Is there a way to avoid currency symbol in the text? if required you can pass the currency symbol as a argument. Please take care of this in all other places too.